ivy/functional/frontends/numpy/sorting_searching_counting/sorting.py

```diff
@@ -33,3 +33,7 @@ def sort_complex(a):
 @to_ivy_arrays_and_back
 def lexsort(keys, /, *, axis=-1):
     return ivy.lexsort(keys, axis=axis)
+
+@to_ivy_arrays_and_back
+def argpartition(a, kth, axis=-1, kind='introselect', order=None):
+    return ivy.argpartition(a, kth, axis=axis, kind=kind)
```

Comment:
ivy has no 'argpartition' function. Please find a different function, or combination of functions, which achieve the same result as numpy.argpartition

Reply:
Hi, @Sam-Armstrong Thank you for your message. I have two questions regarding the argpartition function. Firstly, I would like to know whether I should use the numpy methods or ivy methods when combining different functions to implement the argpartition method. Secondly, I would appreciate it if you could provide me with information on where to find the implemented methods of ivy.

Reply:
@MelatKebedeAbraham yes, all the functions should be ivy functions as this is part of the frontend api. I'd recommend reading through these two pages in the docs for more info about this 😀:

https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends.html
https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html

To your second question, all the functions in the ivy functional api are contained within the files in this directory and its sub-directories: https://github.com/unifyai/ivy/tree/master/ivy/functional/ivy

Feel free to let me know if you have more questions!

Reply:
@Sam-Armstrong Thank you for your guidance and support. I appreciate your help and will definitely reach out if I have any further questions.